README.md

```diff
@@ -4,38 +4,37 @@
 [![Contribute](https://img.shields.io/static/v1?label=Contribute&message=💪)](#contribute)
 [![Release](https://img.shields.io/github/release/opengisch/QField.svg?label=Release)](https://github.com/opengisch/QField/releases)
 [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/8392/badge)](https://www.bestpractices.dev/projects/8392)
-
+[![Digital Public Good](https://img.shields.io/badge/Digital%20Public%20Good-verified-brightgreen)](https://www.digitalpublicgoods.net/r/qfield)
+[![QFieldCloud](https://img.shields.io/badge/QFieldCloud-GitHub-blue)](https://github.com/opengisch/QFieldCloud)
 
 # QField for QGIS
 
-A simplified touch optimized interface for QGIS in the field.
-
-[![QField](https://i0.wp.com/www.opengis.ch/wp-content/uploads/2022/11/QField_App-Medienmitteilung_AppAward2022.jpg?w=1024&ssl=1)](https://qfield.org)
-
-## Get it
-
-Check the [documentation](https://docs.qfield.org/)
-
-### Android
+A simplified touch-optimized interface for QGIS in the field.
 
-[![Get it on Google Play](https://qfield.org/images/play_store.png)](https://play.google.com/store/apps/details?id=ch.opengis.qfield)
+[![header](https://github.com/user-attachments/assets/88771ae0-3701-4cf4-8d8c-cd295c0831b1)](https://qfield.org)
```

Comment:
what is "header" @mbernasocchi ?

Reply:
the big image replacing this
![image](https://github.com/user-attachments/assets/ec7ed4de-80db-42b2-99c3-213a0d0a9544)